semcore/wizard/src/Wizard.jsx

```diff
@@ -124,7 +171,8 @@ function Stepper(props) {
   return sstyled(styles)(
     <SStepper
       render={Box}
-      role='menuitem'
+      role='tab'
+      aria-controls={active ? `${uid}-content-${step}` : undefined}
```

Comment:
sorry, I just realized we should've replaced `aria-current` with `aria-selected`! `aria-selected` should be used in tablists